stock_move_backdating/models/stock_inventory_line.py

```diff
@@ -8,8 +8,8 @@ class StockInventoryLine(models.Model):
     _inherit = "stock.inventory.line"
 
     def _get_move_values(self, qty, location_id, location_dest_id, out):
-        res = super()._get_move_values(qty, location_id, location_dest_id, out)
         self.ensure_one()
+        res = super()._get_move_values(qty, location_id, location_dest_id, out)
```

Comment:
@toita86 As super method calls ensure_one(), you can remove it here.

Reply:
Done